public/assets/js/new_recipe.js

```diff
@@ -1,39 +1,37 @@
-$(document).ready(() => {
-  const title = $("#newTitle").val().trim();
-  const ingredients = $("#newIngredients").val().trim();
-  const method = $("#newMethod").val().trim();
-  const credit = $("#newCredit").val().trim();
-  const source = $("#newSource").val().trim();
-  const addImage = $("#newImage").val().trim();
-  const alertText = $("#alert-text");
+// $(document).ready(() => {
 
-  function newRecipe() {
-    $.ajax({
-      method: "POST",
-      url: "/api/add",
-      data: {
-        title, ingredients, method, credit, source, addImage,
-      },
-    }).then((res) => {
-      if (res) {
-        alertText.text("New Recipe added Successfully");
-      }
-    });
-  }
+const title = $("#newTitle");
+const ingredients = $("#newIngredients");
+const method = $("#newMethod");
+const credit = $("#newCredit");
+const source = $("#newSource");
+const addImage = $("#newImage");
+const alertText = $("#alert-text");
+// const submit = $("#submit");
 
-  function runCheck() {
-    if (title !== "" && ingredients !== "" && method !== "") {
-      newRecipe();
-    } else {
+$("#submit").on("click", (event) => {
+  event.preventDefault();
+  // function newRecipe() {
+  const data = {
+    title: title.val().trim(),
+    ingredients: ingredients.val().trim(),
+    method: method.val().trim(),
+    creditTo: credit.val().trim(),
+    source: source.val().trim(),
+    photo: addImage.val().trim(),
+  };
+  // eslint-disable-next-line no-console
+  console.log(data);
+  $.ajax({
+    method: "POST",
+    url: "/recipe/add",
+    data,
+  }).then((res) => {
+    if (res) {
+      alertText.text("Your new recipe has been added!");
```

Comment:
You know. you can use my alertUser function in /public/assets/js/common.js. See how to use it in login.js